File: CursoPOO/JS/driver.js
class Driver extends Account {
    constructor(id,name,document,email,password){
        super(id,name,document,email,password)
    }

    printDataUser(){
        console.log(`Name: ${this.name}`)
        console.log(`Document: ${this.document}`)
        console.log(`Email: ${this.email}`)
        console.log(`Password: ${this.password}`)
    }
}

File: CursoPOO/Java/Driver.java
class Driver extends Account {

    public Driver(Integer id,String name, String document, String email, String password) {
        super(id ,name ,document, email, password);
    }
    void printDataUser(){
        System.out.println("Id: "+ id +"Name: "+ name +"Document driver: "+ document +"Email: "+ email +"Password: "+ password);
    }
    
}
